Sources/SKTestSupport/IndexedSingleSwiftFileWorkspace.swift

```diff
@@ -46,7 +53,7 @@ public struct IndexedSingleSwiftFileWorkspace {
     var compilerArguments: [String] = [
       testFileURL.path,
       "-index-store-path", indexURL.path,
-      "-index-ignore-system-modules",
```

Comment:
Did we end up needing system modules in a test? Could pass that in to avoid the extra time when it isn't needed.

Reply:
Good idea.